app/javascript/react/screens/App/Mappings/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -117,9 +117,12 @@ class InfrastructureMappingsList extends React.Component {
       <React.Fragment>
         <ListViewToolbar
           filterTypes={INFRA_MAPPINGS_FILTER_TYPES}
+          defaultFilterTypeIndex={0}
           sortFields={INFRA_MAPPINGS_SORT_FIELDS}
+          defaultSortTypeIndex={1}
```

Comment:
Not a concern for the issue at hand, but since you're already touching this, can we change the default sort criterion to `name`?  I was going to add this as part of #733 :)

Reply:
Sure :) done in c3a25c620f006e910da3bd387f29a7b7ce8063d4